### PR TITLE
Fix document

### PR DESCRIPTION
**Pull request by zmunm**
I fix some documentation.

By the way, it would be nice if lint of non-compliant blocks and compliant blocks could also be checked in the ci process. At least itself
If possible, I will open it as a separate issue.

### CODE REVIEW

File: detekt-rules-errorprone/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/bugs/ExitOutsideMain.kt [hunk 1]
```diff
@@ -23,23 +23,23 @@ import org.jetbrains.kotlin.resolve.descriptorUtil.fqNameOrNull
...
- *     exitProcess(2)
- *   } else {
- *     exitProcess(0)
- *   }
+ *     val result = doWork()
+ *     if (result == FAILURE) {
+ *       exitProcess(2)
+ *     } else {
+ *       exitProcess(0)
+ *     }
```
Comment (schalkms) [inline detekt-rules-errorprone/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/bugs/ExitOutsideMain.kt:31]:
```suggestion
 *     val result = doWork()
 *     if (result == FAILURE) {
 *         exitProcess(2)
 *     } else {
 *         exitProcess(0)
 *     }
```

File: detekt-rules-errorprone/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/bugs/ExitOutsideMain.kt [hunk 2]
```diff
@@ -23,23 +23,23 @@ import org.jetbrains.kotlin.resolve.descriptorUtil.fqNameOrNull
...
```
Comment (schalkms) [inline detekt-rules-errorprone/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/bugs/ExitOutsideMain.kt:42]:
```suggestion
 *     val result = doWork()
 *     if (result == FAILURE) {
 *         exitProcess(2)
 *     } else {
 *         exitProcess(0)
 *     }
```

**Review by schalkms — APPROVED** [review]

File: detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/ModifierOrder.kt [hunk 3]
```diff
@@ -43,11 +43,11 @@ import org.jetbrains.kotlin.psi.psiUtil.allChildren
  * taken from: http://kotlinlang.org/docs/reference/coding-conventions.html#modifiers
  *
  * <noncompliant>
- * lateinit internal private val str: String
+ * lateinit internal val str: String
```
Comment (BraisGabin) [inline detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/ModifierOrder.kt:46]:
```suggestion
 * lateinit internal var str: String
```

File: detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/ModifierOrder.kt [hunk 4]
```diff
@@ -43,11 +43,11 @@ import org.jetbrains.kotlin.psi.psiUtil.allChildren
...
  * </noncompliant>
  *
  * <compliant>
- * private internal lateinit val str: String
+ * internal lateinit val str: String
```
Comment (BraisGabin) [inline detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/ModifierOrder.kt:50]:
```suggestion
 * internal lateinit var str: String
```

**Review by BraisGabin — APPROVED** [review]

**Comment by BraisGabin** [comment]
Thanks for the pr! I think that compile the snippets should not be too complex. But I'm not sure how much false positives we will have.

**Pull request merged**